lib/engine/game/g_18_usa/step/buy_sell_par_shares.rb

```diff
@@ -137,6 +137,16 @@ def process_assign(action)
               company.close!
             end
 
+            corporation_hex = corporation.tokens.first.hex
+
+            if company.id == 'P10' && @game.potential_metropolitan_hexes.include?(corporation_hex.id) &&
+                @game.active_metropolitan_hexes.none? { |hex| hex.id == corporation_hex.id }
+              @game.log << "#{company.name} turns #{corporation_hex.location_name} into a metropolis"
+              puts 'metttttro'
```

Comment:
Remove debug

Reply:
Forgot visual studio sometimes refuses to save if the filesystem version changes (like if you run rubocop -A..), changed